cql3/statements/role-management-statements.cc

```diff
@@ -68,11 +68,23 @@ using result_message = cql_transport::messages::result_message;
 using result_message_ptr = ::shared_ptr<result_message>;
 
 static auth::authentication_options extract_authentication_options(const cql3::role_options& options) {
+    static std::string_view allowed_role_options[] = {"read_timeout", "write_timeout"};
```

Comment:
Ah, you overlap role-default-session options with custom authenticator options. I don't know if that's good. Maybe they should have a separate namespace.

Reply:
(but maybe not)

Reply:
Yeah, I let them coexist here, assuming that it's unlikely that a custom authenticator option would be called `read_timeout` - especially that currently there are 0 authenticators which actually use these options at all. So it's a compromise between totally hijacking the options and introducing a special syntax.